lib/log.js

```diff
@@ -246,6 +249,32 @@ function optionallySetService (data, request) {
   }
 }
 
+const FLOW_EVENT_ROUTES = new Set([
+  '/account/create',
+  '/account/destroy',
+  '/account/keys',
+  '/account/login',
+  '/account/login/send_unblock_code',
+  '/account/reset',
+  '/recovery_email/resend_code',
+  '/recovery_email/verify_code'
+])
+
+function logRouteFlowEvent(log, request, response) {
+  const path = request.path
+  if (!FLOW_EVENT_ROUTES.has(path)) {
+    return
+  }
+  const code = response.statusCode || response.output.statusCode
```

Comment:
Ah, this line was just basically copied from the `summary` method. I believe the fallback is that if `response instanceof Error`, it will have the status code on the output property.